src/app/dashboard/types.ts

```diff
@@ -1,6 +1,13 @@
 import {TaskStatus } from '@aneoconsultingfr/armonik.api.angular';
 
-
+export type Line = {
+  name: string,
+  interval: number,
+  hideGroupsHeader: boolean,
+  filters: [],
```

Comment:
You can add some comments in your code to easily understand why there is no types here and to know what's next.

```suggestion
  // TODO: waiting for full filters before adding correct type.
  filters: [],
```